js_modules/dagit/packages/eslint-config/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@dagster-io/eslint-config",
+  "version": "1.0.0",
+  "description": "Shared eslint configuration for @dagster-io",
+  "license": "Apache-2.0",
+  "main": "index.js",
+  "peerDependencies": {
+    "eslint": "^8.6.0"
+  },
+  "devDependencies": {
+    "@types/eslint": "^8",
+    "@typescript-eslint/eslint-plugin": "5.8.0",
+    "@typescript-eslint/parser": "5.8.0",
+    "eslint-config-prettier": "8.3.0",
+    "eslint-plugin-graphql": "^4.0.0",
+    "eslint-plugin-import": "2.26.0",
+    "eslint-plugin-jsx-a11y": "6.5.1",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "7.27.1",
+    "eslint-plugin-react-hooks": "4.3.0",
+    "graphql": "^15.5.0",
+    "prettier": "2.6.2"
```

Comment:
I wonder if `prettier` should technically a peer dep? I guess it's not required that it's used in the parent project, but it'd be nice if the version was loose here (eg: 2.x.x) and specified in the parent so we don't get weird alignment issues if we upgrade it?

Reply:
Yeah, not a bad idea.